src/components/link/link.ts

```diff
@@ -116,6 +116,10 @@ export class MLink extends ModulVue {
         return this.mode === MLinkMode.RouterLink;
     }
 
+    private get withStroke(): boolean {
+        return (this.icon && (!this.iconName || this.iconName === '')) || this.iconName.includes('m-svg__chevron');
```

Comment:
Je crois qu'on peut retirer la condition `this.iconName === ''`. À tester si ça fonctionne toujours.

**Exemple**
```
return (this.icon && !this.iconName) || this.iconName.includes('m-svg__chevron');
```